### PR TITLE
add QazAI customer blog

### PR DESCRIPTION
**Pull request by brianzhan**

### CODE REVIEW

File: _posts/2022-06-02-building-a-modern-data-stack-for-qaz-ai.md [hunk 1]
```diff
@@ -0,0 +1,368 @@
...
+as
+select *
+from tpch.sf3000.partsupp
+
+```
+
+Then, we try to join all three of these tables as it is shown in the ER diagram.
+Let’s make it more challenging by turning off one of the workers, which should
+result in a query failure. To enable the automatic query rerun of the failed one
+we will set `retry_policy=QUERY` in `config. properties`.
```
Comment (mosabua) [inline _posts/2022-06-02-building-a-modern-data-stack-for-qaz-ai.md:304]: we set

File: _posts/2022-06-02-building-a-modern-data-stack-for-qaz-ai.md [hunk 2]
```diff
@@ -0,0 +1,368 @@
...
+left join iceberg2.ice.tpch_sf100000_partsupp psupp on litem.partkey = psupp.partkey and litem.suppkey = psupp.suppkey 
+left join iceberg2.ice.tpch_sf100000_orders ord on litem.orderkey = ord.orderkey 
+```
+
+The query has been completed in 4 hours. Also, at query processing, worker 22
+has been turned off. The query has been automatically started over and completed
+successfully. At the query processing, three tables have been joined (_the
+triple join_): 18 billion rows x 2,4 billion rows x 450 million rows. 
+
+
```
Comment (mosabua) [inline _posts/2022-06-02-building-a-modern-data-stack-for-qaz-ai.md:340]: Remove redundant empty lines

File: _posts/2022-06-02-building-a-modern-data-stack-for-qaz-ai.md [hunk 3]
```diff
@@ -0,0 +1,368 @@
...
+As a result we will achieve faster pipelines. 
+
+A huge thanks to the Trino development team and the Trino community for an
+excellent product, which I enjoy using and allows me to go beyond conventional
+usage patterns.
+
+If you are looking for help building your data warehouse, or if you’re
+interested in joining us at QazAI, feel free to reach out to me at
+kuspayev@aifund.kz
```
Comment (mosabua) [inline _posts/2022-06-02-building-a-modern-data-stack-for-qaz-ai.md:368]: Not sure we should do that, lets remove the email.. instead maybe mention the username on the trino slack and get users to ask you there .. also maybe refer to the project-tardigrade channel

**Review by mosabua — CHANGES_REQUESTED** [review]

**Comment by mosabua** [comment]
@brianzhan are you addressing the review notes?

File: _posts/2022-06-02-building-a-modern-data-stack-for-qaz-ai.md [hunk 4]
```diff
@@ -0,0 +1,368 @@
...
+</p>
+
+
+
+For instance, we create the database shown above in PostgreSQL and work with the _rental_ table.
+
+First, we will move the table from PostgreSQL to our warehouse in HDFS and Hive.
+
+```sql
+create table hive.test.dvd_rental
```
[the remark this answers is not in the document]
Comment (brianzhan) [reply _posts/2022-06-02-building-a-modern-data-stack-for-qaz-ai.md:90]: I don't think SQL should necessarily be uppercase- we actually turn things to lowercase with the Galaxy formatter

[the remark this answers is not in the document]
Comment (mosabua) [reply _posts/2022-06-02-building-a-modern-data-stack-for-qaz-ai.md:86]: The standard we follow in all Trino docs is uppercase .. please just follow it for readability reasons.

File: _posts/2022-06-02-building-a-modern-data-stack-for-qaz-ai.md [hunk 5]
```diff
@@ -0,0 +1,350 @@
+---
+layout: post
+title:  "Building A Modern Data Stack for QazAI"
+author: "Baurzhan Kuspayev" 
+excerpt_separator: <!--more-->
+---
+
+At QazAI, we build data lakes as a service for companies.  In the original
+architecture, we get raw data in S3, transform the S3 data with Hive, and then
+delivered the data to business units via our datamart built on Clickhouse (for optimal delivery speeds). Over time, we were dragged down by the slower speeds and high costs of running Hive, and started shopping for a faster and cheaper open source engine to do our ETL data transformations.
```
Comment (mosabua) [inline _posts/2022-06-02-building-a-modern-data-stack-for-qaz-ai.md:10]: wrap please

**Review by mosabua — APPROVED** [review]
I think this is good to go now. We should probably update the date (filename) just before we merge to whatever day ...

**Review by arhimondr — APPROVED** [review]
@brianzhan please update the date and it should be ready to go

**Pull request merged**